### PR TITLE
Fixed typo in State-Machines wiki page

### PR DESCRIPTION
**Pull request by UnAfraid**
As title says, there was duplication of the class ReplyFlowBot declaration.

### CODE REVIEW

**Comment by rubenlagus** [comment]
# [Codecov](https://codecov.io/gh/rubenlagus/TelegramBots/pull/732?src=pr&el=h1) Report
> Merging [#732](https://codecov.io/gh/rubenlagus/TelegramBots/pull/732?src=pr&el=desc) into [master](https://codecov.io/gh/rubenlagus/TelegramBots/commit/caf3659de90d35f2b94c56f38f09c3d35c28c8bb&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/rubenlagus/TelegramBots/pull/732?src=pr&el=tree)

```diff
@@            Coverage Diff            @@
##             master     #732   +/-   ##
=========================================
  Coverage     13.25%   13.25%           
  Complexity      427      427           
=========================================
  Files           238      238           
  Lines          7372     7372           
  Branches        686      686           
=========================================
  Hits            977      977           
  Misses         6308     6308           
  Partials         87       87           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/rubenlagus/TelegramBots/pull/732?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/rubenlagus/TelegramBots/pull/732?src=pr&el=footer). Last update [caf3659...37ab80a](https://codecov.io/gh/rubenlagus/TelegramBots/pull/732?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

File: TelegramBots.wiki/abilities/State-Machines.md [hunk 1]
```diff
@@ -70,7 +70,6 @@ Now, after your naughty bot retaliates, the user can say "go left or else" to fo
 
 ## Complete Example
 ```java
-public static class ReplyFlowBot extends AbilityBot {
   public class ReplyFlowBot extends AbilityBot {
     public ReplyFlowBot(String botToken, String botUsername) {
       super(botToken, botUsername);
```
Comment (addo37) [inline TelegramBots.wiki/abilities/State-Machines.md]: You could fix the indentation. Thanks for pointing this out tho!

Comment (UnAfraid) [reply TelegramBots.wiki/abilities/State-Machines.md]: Done, indented

Comment (addo37) [reply TelegramBots.wiki/abilities/State-Machines.md]: Thanks!

**Comment by addo37** [comment]
Just realized, this PR should be against the dev branch not master.

**Comment by rubenlagus** [comment]
Closing pull request against master

**Comment by UnAfraid** [comment]
Well we could've just updated the base branch, anyway i've open another PR https://github.com/rubenlagus/TelegramBots/pull/752

**Pull request closed without merging**